build/yamls/base/conf/antrea-controller.conf

```diff
@@ -53,3 +53,22 @@ featureGates:
 # API groups. After adding the annotation, legacy CRDs can be deleted safely without impacting
 # new CRDs.
 #legacyCRDMirroring: true
+
+# Enable the integrated node IPAM controller within the Antrea controller.
+#enableNodeIPAMController: false
+
+# Cluster IP CIDR ranges for Pods. Required when enableNodeIPAMController is enabled, and should be set with
+# the value value specified by --cluster-cidr for kube-apiserver.
+#clusterCIDRs: 172.18.0.0/16
+
+# Service IP CIDR range for Services. Required when enableNodeIPAMController is enabled, and should be set with
+# the value value specified by --service-cluster-ip-range for kube-apiserver.
+#serviceCIDR: 172.19.0.0/16
```

Comment:
The serviceCIDR is only used to filter out the range from clusterCIDR if they overlaps. Maybe we could rephrase the comment a little to ask it when necessary, given that usually they don't overlap?

Reply:
Probably could be rephrased as:

specify the server cluster IP CIDR to ensure these addresses are not allocated to Nodes when enableNodeIPAMController is true. It should the same value as the --service-cluster-ip-range kube-apiserver parameter. It's not necessary to specify it when there is no overlap with clusterCIDR

Reply:
Do we need to support Service CIDR overlaps with Pod CIDR? Does K8s NodeIPAM support that?

Reply:
Do we need to support Service CIDR overlaps with Pod CIDR? Does K8s NodeIPAM support that?

Reply:
@jianjuns NodeIPAM [supports overlaps](https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/nodeipam/node_ipam_controller.go#L57) (also supports  a secondary IP).
However we do not have to support this (or could support a secondary CIDR as well).
Any opinion about either of these?

Reply:
I see. I feel for Pod CIDRs, a list of CIDRs is more flexible. But maybe we can support that later? I do not like to require the extra serviceCIDR parameter.

Reply:
The parameter name is clusterCIDRs in plural and so is the functionality :)
The description is misleading though. I'll correct it.